.mega-linter.yml

```diff
@@ -0,0 +1,19 @@
+---
+APPLY_FIXES: none # all, none, or list of linter keys
+FILTER_REGEX_EXCLUDE: (BenchPress/bicep)
+# ENABLE: # If you use ENABLE variable, all other languages/formats/tooling-formats will be disabled by default
+# ENABLE_LINTERS: # If you use ENABLE_LINTERS variable, all other linters will be disabled by default
```

Comment:
Can these two commented lines be removed?

Reply:
Yes, thanks for catching that! Removed